### PR TITLE
Other change requests

### PR DESCRIPTION
**Pull request by Rhian**
other_validation

### Description of change

This adds a new category of validation change request, which has a summary line and a suggestion for what the applicant can do to make the application valid. Also adds a PUT request for the applicant to send their response.

### Story Link

https://trello.com/c/Ieeseilr/404-other-validation-request

### Decisions 

Tests are currently failing because this needs the patch for the overly long controller names that is currently on another branch. I'll rebase once that is merged.

### CODE REVIEW

File: app/controllers/other_validation_change_requests_controller.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  end
+
+  def show
+    @other_validation_change_request = @planning_application.other_validation_change_requests.find(params[:id])
+  end
+
+  def create
+    @other_validation_change_request = @planning_application.other_validation_change_requests.new(other_validation_change_request_params)
+    @other_validation_change_request.user = current_user
+    @current_local_authority = current_local_authority
```
Comment (james) [inline app/controllers/other_validation_change_requests_controller.rb:15]: is this used anywhere?

Comment (Rhian) [reply app/controllers/other_validation_change_requests_controller.rb]: We need the `current_user` but not the `current_local_authority` so I've removed the latter

File: app/views/api/v1/change_requests/index.json.jbuilder [hunk 2]
```diff
@@ -63,4 +63,16 @@ json.data do
...
+  json.other_change_validation_requests @planning_application.other_change_validation_requests do |other_change_validation_request|
+    json.extract! other_change_validation_request,
+                  :id,
+                  :state,
+                  :response_due,
+                  :response,
+                  :summary,
+                  :suggestion,
+                  :days_until_response_due
+    json.type "red_line_boundary_change_request"
```
Comment (james) [inline app/views/api/v1/change_requests/index.json.jbuilder:76]: not a red_line_boundary_change_request :-)

**Review by james — APPROVED** [review]
one small thing, but all good apart from that

File: app/views/api/v1/change_requests/index.json.jbuilder (hunk 2 above)
Comment (Rhian) [reply app/views/api/v1/change_requests/index.json.jbuilder:76]: 🙈 fixing now

**Pull request merged**